conf/ipop-up.config

```diff
@@ -0,0 +1,25 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'iPOP-UP cluster profile, Université paris Cité'
+  config_profile_contact = 'Magali Hennion, bibs@parisepigenetics.com'
+  config_profile_url = 'https://discourse.rpbs.univ-paris-diderot.fr/c/ipop-up'
+}
+
+singularity {
+  // need one image per execution
+  enabled = true
+  runOptions = '-B /shared'
+}
+
+process {
+  executor = 'slurm'
+  queue = 'ipop-up'
+}
+
+params {
+  igenomes_ignore = true
+  // Max resources requested by a normal node on iPOP-UP cluster.
+  max_memory = 100.GB
+  max_cpus = 28
+  max_time = 96.h
+}
```

Comment:
Can this file be renamed to `conf/ipop_up.config`?

Reply:
Done.